docs/bundles/plain.md

```diff
@@ -29,16 +60,125 @@ manifests
 └── deployment.yaml
 ```
 
+For a bundle git repo, any directory that contains only static Kubernetes manifests checked into a git repository
+accessible via a remote URL can be considered a plain bundle and sourced by the plain provisioner. For more information
+on bundles backed by git repositories, see the [git based bundles doc](git-bundles.md).
+
 > Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
 > plain+v0 bundle.
 
 ## Technical Details
 
-* The static manifests must be located in the /manifests directory for the bundle to be a
-valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
-invalid and will not be successfully unpacked onto the cluster.
+* The static manifests must be located in the root-level /manifests directory in a bundle image for the bundle to be a
+  valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
+  invalid and will not be successfully unpacked onto the cluster.
+* For a bundle git repo, the manifests directory can be anywhere in the repository, not just at the root-level. The
```

Comment:
```suggestion
* For a bundle `git` repo, the manifests directory can be anywhere in the repository, not just at the root-level. The
```

Reply:
addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1